controller_interface/src/controller_interface.cpp

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 #include "controller_interface/controller_interface.hpp"
+#include <lifecycle_msgs/msg/state.hpp>
```

Comment:
Why not define this as new type, e.g., ’ControllerState’

Reply:
I disagree. There's no point in introducing more abstractions unless there is a reason. We want to explicitly maintain the lifecycle node states because ideally we'd go back to using them once the service issue is taken care of.